controllers/getCategories-controller.js

```diff
@@ -0,0 +1,7 @@
+const { fetchCategories } = require("../models/fetchCategories-model");
+
+exports.getCategories = (req, res) => {
+  fetchCategories().then(({ rows }) => {
+    res.status(200).send({ categories: rows });
+  });
+};
```

Comment:
we should have a .catch block here, although not loads can go wrong here we would still want one just in case